### PR TITLE
🎉 New Source: Ashby

### PR DESCRIPTION
**Pull request by elliottrabac**
## What
*Describe what the change is solving*

- Connector contest: https://github.com/airbytehq/connector-contest/issues/131

## How
*Describe the solution*

Implementing the Ashby connector with relevant streams using the low Code CDK.

## Recommended reading order
1. `ashby.yaml`
2. `spec.yaml`

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>New Connector</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [ ] `docs/integrations/README.md`
    - [ ] `airbyte-integrations/builds.md`
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)
- [ ] After the connector is published, connector added to connector index as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)
- [ ] Seed specs have been re-generated by building the platform and committing the changes to the seed spec files, as described [here](https://docs.airbyte.io/connector-development#publishing-a-connector)

</details>

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [ ] Create a non-forked branch based on this PR and test the below items on it
- [ ] Build is successful
- [ ] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [ ] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

<details><summary><strong>Connector Generator</strong></summary>

- [ ] Issue acceptance criteria met
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)
- [ ] If adding a new generator, add it to the [list of scaffold modules being tested](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connector-templates/generator/build.gradle#L41)
- [ ] The generator test modules (all connectors with `-scaffold` in their name) have been updated with the latest scaffold by running `./gradlew :airbyte-integrations:connector-templates:generator:testScaffoldTemplates` then checking in your changes
- [ ] Documentation which references the generator is updated as needed

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*
Screenshot 2022-10-22 at 20 09 54

</details>

Check:
Screenshot 2022-10-22 at 20 10 40

Read:
Screenshot 2022-10-22 at 20 10 27

I tested it on a GCP VM as well:
Screenshot 2022-10-22 at 20 11 44

### CODE REVIEW

File: airbyte-integrations/builds.md [hunk 1]
```diff
@@ -2,164 +2,165 @@
 
 # Sources
 
-|name |status |
-| :--- | :--- |
-| 3PL Central  | [![source-amazon-seller-partner](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-tplcentral%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-tplcentral) |
-| Adjust  | [![source-adjust](https://img.shields.io/endpoint?url=https%3A%2F%2Fdnsgjos7lj2fu.cloudfront.net%2Ftests%2Fsummary%2Fsource-adjust%2Fbadge.json)](https://dnsgjos7lj2fu.cloudfront.net/tests/summary/source-adjust) |
```
Comment (marcosmarxm) [inline airbyte-integrations/builds.md:8]: Check this change.

File: airbyte-integrations/connectors/source-ashby/integration_tests/configured_catalog.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "users",
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-ashby/integration_tests/configured_catalog.json:5]: you must add all streams to run the integration tests.

File: docs/integrations/README.md [hunk 3]
```diff
@@ -14,220 +14,221 @@ For more information about the grading system, see [Product Release Stages](http
 
 ## Sources
 
-| Connector                                                                                   | Product Release Stage| Available in Cloud? |
-|:--------------------------------------------------------------------------------------------| :------------------- | :------------------ |
-| [3PL Central](sources/tplcentral.md)                                                        | Alpha                | No                  |
-| [Adjust](sources/adjust.md)                                                                 | Alpha                | No                  |
-| [Airtable](sources/airtable.md)                                                             | Alpha                | Yes                 |
-| [AlloyDB](sources/alloydb.md)                                                               | Generally Available  | Yes                 |
-| [Amazon Ads](sources/amazon-ads.md)                                                         | Generally Available  | Yes                 |
```
Comment (marcosmarxm) [inline docs/integrations/README.md:23]: Check this change.

File: airbyte-integrations/connectors/source-ashby/source_ashby/spec.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  title: Ashby Spec
+  type: object
+  required:
+    - api_key
+  additionalProperties: true
+  properties:
+    api_key:
+      type: string
+      description: The Ashby API Key.
+      airbyte_secret: true
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-ashby/source_ashby/spec.yaml:13]: What about adding a start date parameter to retrieve data only after a period? I saw endpoints allow this feature: 
image

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
A few comments, please run tests with all streams and check the comments. @elliottrabac do you have a test credentials to us run the tests too?

File: airbyte-integrations/builds.md (hunk 1 above)
Comment (elliottrabac) [reply airbyte-integrations/builds.md]: My bad. Reverted and changed without yml formatting.

File: docs/integrations/README.md (hunk 3 above)
Comment (elliottrabac) [reply docs/integrations/README.md]: Reverted and changed without yml formatting.

File: airbyte-integrations/connectors/source-ashby/integration_tests/configured_catalog.json (hunk 2 above)
Comment (elliottrabac) [reply airbyte-integrations/connectors/source-ashby/integration_tests/configured_catalog.json:122]: ✅

File: airbyte-integrations/connectors/source-ashby/source_ashby/spec.yaml (hunk 4 above)
Comment (elliottrabac) [reply airbyte-integrations/connectors/source-ashby/source_ashby/spec.yaml]: createdAfter is available for application and interview_schedules.

I added the filter for them ✅

**Comment by elliottrabac** [comment]
I made some changes based on the comments 👍 

The filter using createdAfter is also working well:
Screenshot 2022-10-25 at 07 04 58
Screenshot 2022-10-25 at 07 04 16

@marcosmarxm I don't have a sandbox for this tool, and we can't set up a free-trial :/
What can I do to help here?

**Comment by marcosmarxm** [comment]
No problem @elliottrabac I'll review your contribution tomorrow again.

**Review by marcosmarxm — APPROVED** [review]
Congrats @elliottrabac !!! :) I'll publish later today.

**Comment by RealChrisSean** [comment]
Congrats!

**Pull request merged**